packages/devtools_testing/fixtures/provider_app/.gitignore

```diff
@@ -0,0 +1,46 @@
+# Miscellaneous
```

Comment:
move all these fixtures to a separate CL and I'll give them a quick LGTM. That will also help move the main CL down to a reviewable size.